Resources/Prototypes/_Mono/Entities/Objects/Weapons/Guns/Rifles/rifles.yml

```
@@ -0,0 +1,61 @@
+- type: entity
+  name: MR-8A AR
+  parent: [BaseWeaponRifle, BaseGunWieldable, BaseC3Contraband]
+  id: WeaponRifleMR8
+  description: An advanced military firearm, firing .30 rifle with astonishing accuracy compared to other rifles.
+  components:
+  - type: Sprite
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)